src/components/NotificationsDrawer/NotificationItem.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import {
+  Checkbox,
+  Dropdown,
+  DropdownItem,
+  Label,
+  MenuToggle,
+  MenuToggleElement,
+  NotificationDrawerList,
+  NotificationDrawerListItem,
+  NotificationDrawerListItemBody,
+  NotificationDrawerListItemHeader,
+} from '@patternfly/react-core';
```

Comment:
Same here, you want to use absolute import paths as of PF5. Improves build sizes.